specificatie/partner.yaml

```diff
@@ -131,3 +133,23 @@ components:
       properties:
         datum:
           $ref: 'datum.yaml#/components/schemas/GbaDatum'
+    OntbindingHuwelijkPartnerschap:
+      type: object
+      description: |
+        Gegevens over de ontbinding van het huwelijk of het geregistreerd partnerschap.
+        * **datum** : De datum waarop het huwelijk of het partnerschap is ontbonden.
+      properties:
```

Comment:
In de partnerhistorie hebben we nu ontbindingHuwelijkPartnerschap.indicatieHuwelijkPartnerschapBeeindigd opgenomen.

Reply:
Is jouw voorstel om ook hier ontbindingHuwelijkPartnerschap.indicatieHuwelijkPartnerschapBeeindigd op te nemen en niet de datum ?